`General`: Add troubleshooting page to the documentation (#4598)
ls1intum/Artemis: docs/admin/troubleshooting.rst

Troubleshooting
===============

> Repository download  of large courses fails
> -------------------------------------------

Reviewer: ```suggestion
Repository download of large courses fails
------------------------------------------
```

The most likely issue is that the configured timeouts on the reverse proxy are too low. 

> If you set up your Artemis instance with the `Ansible Role (>=v0.1.1) <https://github.com/ls1intum/artemis-ansible-collection>`__, you can set the timeout values with the following variables:

Reviewer: You could also mention here that the same configuration parameters can be used in a manually set up nginx, as I assume this is the most common setup at the moment.

Troubleshooting
===============

Repository download  of large courses fails
-------------------------------------------

> The most likely issue is that the configured timeouts on the reverse proxy are to low. 

Reviewer 2: ```suggestion
The most likely issue is that the configured timeouts on the reverse proxy are too low. 
```

> If you set up your Artemis instance with the `Ansible Role (>=v0.1.1) <https://github.com/ls1intum/artemis-ansible-collection>`__, 
> you can set the timeout values with the following varaibles:

Reviewer 2: ```suggestion
If you set up your Artemis instance with the `Ansible Role (>=v0.1.1) <https://github.com/ls1intum/artemis-ansible-collection>`__, you can set the timeout values with the following variables:
```

Troubleshooting
===============

Repository download of large courses fails
------------------------------------------

The most likely issue is that the configured timeouts on the reverse proxy are too low. 

If you set up your Artemis instance with the `Ansible Role (>=v0.1.1) <https://github.com/ls1intum/artemis-ansible-collection>`__, you can set the timeout values with the following variables:

.. code:: yaml

   proxy_send_timeout: "900s"
   proxy_read_timeout: "900s"
   fastcgi_send_timeout: "900s"
   fastcgi_read_timeout: "900s"

> If you conrigured the ngnix reverse proxy by hand, you have to adapt the following variables in your nginx configurations: 

Reviewer: ```suggestion
If you configured the nginx reverse proxy by hand, you have to adapt the following variables in your nginx configurations:

```

Author: 🙈
